widget/form_test.go

```diff
@@ -72,7 +72,8 @@ func TestForm_AddRemoveButton(t *testing.T) {
 			assert.Fail(t, "Timed out waiting for submit button tap")
 		}
 	}()
-	assert.Equal(t, 1, scount)
+	assert.Equal(t, 1, scount) // because we tapped submit
```

Comment:
I suggest to use `assert`’s message instead of adding comments:
```suggestion
	assert.Equal(t, 1, scount, "tapping submit should increase scount")
```

Reply:
:+1: